lib/ansible/modules/cloud/openstack/os_security_group.py

```diff
@@ -117,7 +117,8 @@ def main():
 
     try:
         cloud = shade.openstack_cloud(**module.params)
-        secgroup = cloud.get_security_group(name)
+        filters = {'tenant_id': cloud.keystone_session.get_project_id()}
```

Comment:
This should use cloud.current_project_id property instead, which equates to the same call, but abstracts away the need to interact with the keystone session which someone really shouldn't ever need to do. This property was added in shade 1.13.0, so maybe a min version check here, too?